### ISSUE TITLE
Voice input for search

### ISSUE THREAD

**Issue by areeshatariq** [mozilla-mobile/outreachy-UX-2020#68]
### What is the problem you are trying to solve?
[#1216](https://github.com/mozilla-mobile/fenix/issues/1216) Voice input for search
### Why is it a problem?
As part of efforts toward accessibility, this feature needs to be added to the Fenix application. Certain categories of people are excluded from using the application because they are unable to use the interface of the application. Voice inputs can be used to control the application, which can thus open up the possibility for people with difficulty to be included. 
### How would you solve the problem? What is your design proposal?
Voice input for search feature will be included in the Fenix mobile application. To make the application design more _inclusive,_ more features can be added that can enable to operate the application completely through voice.  

## Design
Voice input icon will be placed on the search bar. The voice recognition system will start listening when the user will press the icon.
<p align="center"><img src="https://user-images.githubusercontent.com/24654457/77862898-cc9f9680-7237-11ea-85c2-98299b1fe498.jpg" width="50%" height="50%"></p>

### How would you measure your designs effectiveness?

- By looking at user's reviews. They can help to improve the design iteratively.
- By testing the feature on a group of users. The feature should be easy to use.

**Comment by topotropic** [comment]
Thanks for the proposal!
